src/main/webapp/css/index-page.css

```diff
@@ -16,5 +16,50 @@
 
 /* Change the index page background. */
 body{
-	  background: url("../images/index-background.jpg");     
-}
\ No newline at end of file
+	  background: url("../images/index-1.jpg");
+}
+
+.container {
+    padding: 80px 120px;
+  }
+  .person {
```

Comment:
nit: try to keep the indentation consistent

Reply:
Will keep that in mind. Thanks for the tip  :)